collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java

```diff
@@ -70,8 +71,8 @@ public void resetSettingsTest() throws IOException {
         resetAppState(Collections.singletonList(ResetUtility.ResetAction.RESET_PREFERENCES));
 
         SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(Collect.getInstance());
-        assertEquals(null, settings.getString(PreferenceKeys.KEY_USERNAME, null));
-        assertEquals(null, settings.getString(PreferenceKeys.KEY_PASSWORD, null));
+        assertNull(settings.getString(PreferenceKeys.KEY_USERNAME, null));
```

Comment:
I'm sorry, that was my brain playing tricks on me. I thought that was an `assertNull` argument that made no sense.